web/src/containers/Withdraw/index.tsx

```diff
@@ -109,14 +111,14 @@ export class Withdraw extends React.Component<WithdrawProps, WithdrawState> {
                     </div>
                     <div className="cr-withdraw__divider cr-withdraw__divider-one" />
                     <div className={withdrawAmountClass}>
-                        <CustomInput
+                        <InputWithButton
                             type="number"
+                            value={amount}
                             label={withdrawAmountLabel || 'Withdrawal Amount'}
-                            defaultLabel="Withdrawal Amount"
-                            inputValue={amount}
-                            placeholder={withdrawAmountLabel || 'Amount'}
-                            classNameInput="cr-withdraw__input"
                             handleChangeInput={this.handleChangeInputAmount}
+                            className="cr-withdraw__input"
+                            buttonText="All"
```

Comment:
add 'All' to translation file in src/translations/en.ts and src/custom/translations/ru.ts files